### PR TITLE
COL-346 Return alternate embeddable URLs from preview service

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-346

Right now we're only aware of Google Drive using this embedURL attribute, but it may show up elsewhere.

Goes together with https://github.com/ets-berkeley-edu/suitec/pull/223

### CODE REVIEW

File: lib/preview/link.js [hunk 1]
```diff
@@ -36,6 +37,11 @@ var PreviewConstants = require('./constants');
 var PreviewImage = require('./image');
 var Result = require('./model').Result;
 
+// URLs matching any of these regexes will be cheched for an "embedURL" meta tag.
+var CHECK_EMBED_URL_REGEXES = [
+  /^http(s)?:\/\/drive\.google\.com\//
```
Comment (johncrossman) [inline lib/preview/link.js]: @paulkerschen should we anticipate `docs.google.com`?

Comment (paulkerschen) [inline lib/preview/link.js]: @johncrossman Good thought - I just checked and confirmed that `embedURL` appears in Docs as well. Will amend.

**Comment by johncrossman** [comment]
Looks good, @paulkerschen   Waiting on your reply to one question.

**Comment by paulkerschen** [comment]
Thanks @johncrossman - amended to include docs.google.com.

**Pull request merged**